### DISCUSSION TITLE
Converting tests from October / Winter v1 & v1.1?

### DISCUSSION THREAD

**Discussion by samcoppock** [wintercms/winter#1177] [Q&A] [answered]
october V1 and previous winter versions used "../../../tests/bootstrap.php" in the tests.

The file was found in the <project root>/tests folder and was referenced in the phpunit.xml file

winter 1.2 no longer has the <project root>/tests folder and the phpunit file does not reference "../../../tests/bootstrap.php"
it does however reference "./modules/system/tests/bootstrap/app.php"

Does app.php completely replace bootstrap.php?

**Comment by LukeTowers** [comment, marked as answer]
Yes. See https://github.com/wintercms/meta/blob/master/l9-upgrade-notes.md#upgrade-unit-testing for more details on converting.
